docker/nginx/wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
If I'm right this script is taken from https://github.com/vishnubob/wait-for-it - the backlinks to the project are missing here.

Reply:
You'e totally right and I should have added the reference. I'll take care of it.

Reply:
Added a comment on line 2 that states the source and license of the script.